### PR TITLE
4808 update login error component

### LINKED ISSUE #4808: Update login error component
## Is your feature request related to a problem? Please describe 👀

The login error message doesn't look good and hides extra detail behind a tooltip, which is hard to use on android.
I realise that this is similar to #2654 and #3815 and #1917 and #1770 and #2945
This issue though is about updating the style of the message only. Those issues are more about the content of the error displayed.

## Describe the solution you'd like 🎁
Can you update the login error style to look like this please:

telegram-cloud-photo-size-1-5138705203619540305-y

and if there's more details, show them with an expanding panel:

telegram-cloud-photo-size-1-5140915754567314880-y

telegram-cloud-photo-size-1-5140915754567314881-y

### PR DESCRIPTION
**Pull request by roxy-dao**
Fixes #4808

# 👩🏻‍💻 What does this PR do?
Changes the error component in the login screen to have a text expansion instead of using tooltip. Have made this into its own separate component since the ErrorWithDetails component is used by other parts of the app. 

Icon looks a bit different since it didn't mesh well with the background.... lemme know what you think (: 

https://github.com/user-attachments/assets/43db8a31-1cb7-4aed-8679-681212866bc5

Screenshot 2024-10-08 at 11 29 05 AM

## 💌 Any notes for the reviewer?
Another issue to review existing tooltip error information and change existing ErrorWithDetails component? 

# 🧪 Testing

- [ ] Prompt some kind of error in login page (by disconnecting server or entering wrong password 3 times)
- [ ] See new error component
- [ ] Click `More information` 

# 📃 Documentation

- [ ] **Part of an epic**: documentation will be completed for the feature as a whole
- [ ] **No documentation required**: no user facing changes or a bug fix which isn't a change in behaviour
- [x] **These areas should be updated or checked**: 
  1. Login error screenshots? 
  2.

### CODE REVIEW

File: client/packages/common/src/styles/theme.ts [hunk 1]
```diff
@@ -172,7 +172,7 @@ export const themeOptions = {
     outline: { main: '#555770' },
     divider: '#eaeaea',
     drawerDivider: '#eaeaea',
-    error: { main: '#e63535' },
+    error: { main: '#e63535', background: '#ffcdce' },
```
Comment (roxy-dao) [inline client/packages/common/src/styles/theme.ts:175]: Needs a lighter colour for the background... tried to find the closest to the screenshot I can....

File: client/packages/host/src/components/Login/LoginError.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+              display="flex"
+              sx={{
+                cursor: 'pointer',
+                fontSize: 12,
+                color: 'secondary.main',
+              }}
+              onClick={() => setExpand(!expand)}
+            >
+              {t('error.more-info')}
+              {expand ? <ChevronUpIcon /> : <ChevronDownIcon />}
```
Comment (roxy-dao) [inline client/packages/host/src/components/Login/LoginError.tsx:64]: The icons look better than text 😉

**Review by jmbrunskill — APPROVED** [review]
Works great for me, thanks Roxy.

Only slightly weird thing is the up and down chevon look kind of different?

image
vs
image

I notice in the original issue it was a `>` character?
image

Personally I'm happy how it is, but designers might care?

**Comment by mark-prins** [comment]
can we change the chevron please? it troubles me

**Comment by roxy-dao** [comment]
> can we change the chevron please? it troubles me

Both of them?

**Comment by mark-prins** [comment]
Apologies!
Can you please change the chevrons to be as per the mock up in the issue. Should be our `ChevronDown` icon

image

which is rotated -90° when collapsed 🙏 

image

image

**Comment by roxy-dao** [comment]
> Apologies! Can you please change the chevrons to be as per the mock up in the issue. Should be our `ChevronDown` icon
> which is rotated -90° when collapsed 🙏

270 XD

https://github.com/user-attachments/assets/cbd352e0-9985-464a-bb60-337bf70e6798

**Comment by mark-prins** [comment]
oh! too slow. was just loading it up locally to check the sizing - to me the chevron looked too big and I changed to this:

image

added 
```
element.style {
    width: 0.6em;
    margin-top: 0.1em;
    height: 0.6em;
}
``` 
to the svg element. is it possible to squeeze this in somehow?

**Comment by roxy-dao** [comment]
> oh! too slow. was just loading it up locally to check the sizing - to me the chevron looked too big and I changed to this:
> added
> 
> ```
> element.style {
>     width: 0.6em;
>     margin-top: 0.1em;
>     height: 0.6em;
> }
> ```
> 
> to the svg element. is it possible to squeeze this in somehow?

I can do it in my next PR, is that alright?

**Comment by mark-prins** [comment]
thanks!

**Pull request merged**